docs/is_new_stable.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+import packaging.version
```

Comment:
This is a core infrastructure package, used by setuptools, build, etc. So its almost certainly already installed (but we pip install it just in case).